packages/virtualtours/src/css/landing.css

```diff
@@ -54,7 +41,7 @@ header ul li a {
     position: relative;
     margin: 0 15px;
     text-decoration: none;
-    color: #FFFBF5;
+    /* color: #FFFBF5; */
```

Comment:
This is for header's text color. For now, I commented out color so header color is black by default for visibility. Somehow the header text color blends in with background.